components/display_drivers/include/gc9a01.hpp

```diff
@@ -454,17 +415,17 @@ class Gc9a01 {
   }
 
 protected:
-  static display_drivers::write_fn lcd_write_;
-  static display_drivers::send_lines_fn lcd_send_lines_;
-  static gpio_num_t reset_pin_;
-  static gpio_num_t dc_pin_;
-  static int offset_x_;
-  static int offset_y_;
-  static bool mirror_x_;
-  static bool mirror_y_;
-  static bool mirror_portrait_;
-  static bool swap_xy_;
-  static bool swap_color_order_;
-  static std::mutex spi_mutex_;
+  static inline display_drivers::write_command_fn write_command_;
+  static inline display_drivers::send_lines_fn lcd_send_lines_;
+  static inline gpio_num_t reset_pin_;
+  static inline gpio_num_t dc_pin_;
+  static inline int offset_x_;
+  static inline int offset_y_;
+  static inline bool mirror_x_;
+  static inline bool mirror_y_;
+  static inline bool mirror_portrait_;
+  static inline bool swap_xy_;
+  static inline bool swap_color_order_;
+  static inline std::mutex spi_mutex_;
```

Comment:
See my comment above about moving these into the parent class `espp::DisplayDriver`, which of course means they are no longer static.

Reply:
again per the comment above, that will be a future PR.